arise_predictions/auto_model/build_models.py

```diff
@@ -12,6 +12,7 @@
 from sklearn.pipeline import Pipeline
 from sklearn.ensemble import StackingRegressor
 from sklearn.inspection import permutation_importance
+from lineartree import LinearTreeRegressor, LinearForestRegressor, LinearBoostRegressor
```

Comment:
So far we managed to avoid explicit import of estimators - can we do the same here?

Reply:
done